specs/system_info.table

```diff
@@ -0,0 +1,8 @@
+table_name("system_info")
+description("System information for identification.")
+schema([
+    Column("hostname", TEXT, "Hostname"),
+    Column("hostuuid", TEXT, "UUID used by osquery to uniquely identify this host"),
+    Column("cpu_id", TEXT, "CPU ID"),
```

Comment:
Also! We have a cpuid table that returns data from the equivalent ASM/x86 instruction. If it's possible to avoid overloading CPU + ID, let's do it!